sticky/sticky.py

```diff
@@ -109,6 +108,19 @@ async def sticky_toggleheader(self, ctx: commands.Context, true_or_false: bool):
         await self.conf.channel(ctx.channel).header_enabled.set(true_or_false)
         await ctx.tick()
 
+    @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
+    @sticky.command(name="cooldown", aliases=["setcooldown"])
+    async def sticky_cooldown(self, ctx: commands.Context, seconds: int):
+        """Set the cooldown time for reposting sticky messages in this channel.
+        
+        The cooldown must be at least 5 seconds."""
+        if seconds < 5:
+            await ctx.send("The cooldown cannot be set lower than 5 seconds.")
```

Comment:
I'd prefer to keep the previous default of 3 seconds
```suggestion
        The cooldown must be at least 3 seconds."""
        if seconds < 3:
            await ctx.send("The cooldown cannot be set lower than 3 seconds.")
```